docs/content/Contributing/Testing/_index.en.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Testing
+weight: 20
+---
+
+{{< toc >}}
+
+This describes how to run tests locally, and how to write new tests for Eask.
```

Comment:
Do you have any questions or changes you want to this doco page?

Reply:
I've put this page under the "Contributing" section. If that's ok, then I'll remove the existing section in the "Developing Eask" page.

Here's what the side menu looks like for me (ignore missing emoji)
![1743173393](https://github.com/user-attachments/assets/b3ff4dfc-9187-43ff-be73-60ff3c6f7ae5)

Reply:
I'm thinking of moving the entire testing page to Developing Eask’s testing section since there is already one. 🤔 WDYT?

Reply:
Yep that's fine too. The only reason I didn't initially was because it is quite large.

Reply:
Ok done!